app/file.go

```diff
@@ -417,6 +424,25 @@ func getImageOrientation(input io.Reader) (int, error) {
 	}
 }
 
+func genericThumbnailHandler() (*model.AppError) {
```

Comment:
I don't think this is necessary.

Reply:
I did not know that Mattermost app server could find an image that was located in a path outside of the storage model.  This function was to add the thumbnail image to the storage model so that it would be made accessible to any requests to `info.ThumbnailPath`, which = `GENERIC_THUMBNAIL_PATH`.

Reply:
As commented above regarding path of `generic_thumbnail_image.png` but @hmhealey might give better approach to this.